.pipelines/PowerShell-Release-Official.yml

```diff
@@ -270,15 +270,49 @@ extends:
         parameters:
           SkipPSInfraInstallers: ${{ parameters.SkipPSInfraInstallers }}
 
+    - stage: UploadChangelogAndTools
+      displayName: Upload Changelog and Tools
+      dependsOn: BlobPublic
+      jobs:
+      - template: /.pipelines/templates/release-upload-cl-tools.yml@self
+        parameters:
+          depth: 1
+
     - stage: PublishGitHubRelease
       displayName: Publish GitHub Release
       dependsOn: BlobPublic
+      variables:
+        ob_release_environment: Production
       jobs:
       - template: /.pipelines/templates/release-githubtasks.yml@self
+    
+    - stage: PushGitTag
+      displayName: Push Git Tag
+      dependsOn: PublishGitHubRelease
+      jobs:
+      - template: /.pipelines/templates/approvalJob.yml@self
+        parameters:
+          displayName: Push Git Tag
+          jobName: PushGitTag
+          instructions: |
+            Push the git tag to upstream  
+    
+    - stage: MakeDraftPublic
+      displayName: Make Draft Public
+      dependsOn: PushGitTag
+      jobs:
+      - template: /.pipelines/templates/approvalJob.yml@self
+        parameters:
+          displayName: Make Draft Public
+          jobName: DraftPublic
+          instructions: |
+            Make the GitHub Release Draft Public
```

Comment:
These can be combined into a single stage for push and make draft public as both are approval jobs